src/sb_utils.R

```diff
@@ -29,4 +46,42 @@ sb_render_post_xml <- function(sb_id, ..., xml_file = NULL){
   
   sb_replace_files(sb_id = sb_id, xml_file)
   
-}
\ No newline at end of file
+}
+
+# Helper function to create a task_table for the files that need to be pushed to SB
+do_item_replace_tasks <- function(files, sb_id, task_yml, out_dir) {
```

Comment:
Can you skip the `out_dir` argument too and return an object (if needed) or nothing? I'm looking for different ways to shrink the number of arguments that are needed and the amount of files/directories that need to be created

Reply:
Yeah, I struggled with whether or not to return anything here. I suppose the loop_tasks will error if it doesn't work so do we really need that file with the hashes for these "_pushed_to_sb" targets? Feels like no.

Reply:
I think returning a rubble that includes the file names and the time (in UTC) that they were pushed to SB would be useful here. Or that could be a future nice-to-have.

Reply:
Tibble not rubble

Reply:
Seems doable. I will try that now.

Reply:
OK, just tried something that I think would do the trick.